Add KIC 1.2.0 docs (#2712)
Kong/docs.konghq.com: app/kubernetes-ingress-controller/1.2.x/troubleshooting.md

   for sharing with Kong support. `sensitive` dumps the complete configuration
   exactly as it is sent to the Admin API.
1. Check controller logs for the dump location with `kubectl logs PODNAME -c
   ingress-controller | grep "config dumps"`
1. Optionally make a change to a Kubernetes resource that you know will
   reproduce the issue. If you are unsure what change caused the issue
   originally, you can omit this step.
1. Copy dumped configuration out of the controller for local review with
   `kubectl cp PODNAME:/path/to/dump/last_bad.json /tmp/last_bad.json -c
   ingress-controller`. `last_good.json` will also be available if the
   controller successfully applied configuration previously.

Once you have dumped configuration, there are two main approaches to isolate
issues:

- If you know of a specific Kubernetes resource change that reproduces the
  issue, diffing the `last_good.json` and `last_bad.json` will show the change
  the controller is trying to apply unsuccessfully.
- You can apply dumped configuration via the `/config` Admin API endpoint
  (DB-less mode) or using decK (DB-backed mode) to a test instance not managed

>   by the ingress controller. This approach allows for easier review of requests

Reviewer: ```suggestion
  by the ingress controller. This approach lets you review requests
```

  and responses (passing `--verbose 2` to decK will show all requests) and

>   quick addition of debug Kong Lua code when controller requests result in an

Reviewer: ```suggestion
   add debug Kong Lua code when controller requests result in an
```
Should be avoiding terms like "quickly" or "easily" (adverbs as a whole, really) to avoid setting expectations.
